client/src/use-case/implementation/converter/SessionD2BConverter.ts

```diff
@@ -11,4 +11,8 @@ export class SessionD2BConverter {
       data.resultWord
     );
   }
+
+  processResponse(status: number): boolean {
```

Comment:
šitas metodas labai out of place. 
1. Statusą ar pavyko delete Įdėjai į session konverterį
2. tau true ar false jau turėtų grąžinti gateway metodas, o ne mappinti interaktoriaus layeris, nes gw yra atsakingas už rest duomenis (eg. status 204).